doc/particles.md

```diff
@@ -5,85 +5,88 @@ Flame offers a basic, yet robust and extendable particle system. The core concep
 The most basic usage of `Particle` with `BaseGame` would look as following:
 
 ```dart
-import 'package:flame/components/particle_component.dart';
+import 'package:flame/components.dart';
 
 // ... 
 
 game.add(
-    // Wrapping a [Particle] with [ParticleComponent]
-    // which maps [Component] lifecycle hooks to [Particle] ones
-    // and embeds a trigger for removing the component.
+    /// Wrapping a [Particle] with [ParticleComponent]
+    /// which maps [Component] lifecycle hooks to [Particle] ones
+    /// and embeds a trigger for removing the component.
```

Comment:
technically since this is a regular comment (not dartdoc), should be only 2 slashes

Reply:
The reason I added the extra `/` is because it uses Dartdoc notation (`[Reference]`).